components/inputs/test/input-date.visual-diff.js

```diff
@@ -244,24 +244,55 @@ describe('d2l-input-date', () => {
 			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
 		});
 
-		it('open with placeholder', async function() {
-			await helper.open(page, '#no-value');
-			const rect = await helper.getRect(page, '#no-value');
-			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
-			await helper.reset(page, '#no-value');
+		describe('placeholder', () => {
```

Comment:
I moved `placeholder` and `min-max` into their own `describe` so that the afterEach to reset (which is needed otherwise close isn't necessarily complete) is unique to each version of `input-date`. I want to do the same for `basic` but it would have caused too many failures of visual diff which could distract from any real failures.